Feature/poc internal proxy auxiliary creation (#212)
eclipse/kiso-testing: docs/advanced_usage/advanced_config_file.rst

    :language: yaml
    :lines: 28-37

Sharing a communication channel between multiple auxiliaries
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In order to attach a single communication channel to multiple defined auxiliaries, it is sufficient
to use the same channel name for all auxiliaries.

Under the hood, ``pykiso`` will automatically create a :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`

> that will be the only auxiliary having direct access to the communication channel, along with a

Reviewer: ```suggestion
that will have exclusive access to the communication channel.
```

Author: Rephrased it according to your suggestions ;)

> :py:class:`~pykiso.lib.connectors.cc_proxy.CCProxy` for each defined auxiliary. This allows to

Reviewer: ```suggestion
:py:class:`~pykiso.connector.CChannel` will be between the auxiliary you declared in the yaml and the above created proxy.
```
